api/config/compliance/config_request.proto

```diff
@@ -27,7 +27,12 @@ message ConfigRequest {
 			Profiles profiles = 4;
 			Storage storage = 5;
 			Logger logger = 6;
-			Retention retention = 7;
+			// NOTE: retention is no longer configurable via config and is now
+			// managed with gRPC endpoints. The field is not reserved so we can
+			// migrate the initial values from config to to the purge cereal
+			// workflows.
+			// Setting retention values is prevented in the Validate() callback.
+			Retention retention = 7; // deprecated
```

Comment:
`Retention retention = 7 [deprecated=true];`

Reply:
Cool, as far as I know the field tag has no meaning in the go protoc compiler, but the field option seems like a good option.